storage/connectors/nedb.js

```diff
@@ -4,7 +4,7 @@ class nedbAdapter {
   constructor(options) {
     this._nedb = new nedb(
       Object.assign({
-        filename: '.bundle-cache.json',
+        filename: '.bundler-cache',
```

Comment:
tweak to cache file name and removed JSON extension because the cache is [JSONLines](http://jsonlines.org/)